pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -134,6 +140,9 @@ public class ExtensibleLoadManagerImpl implements ExtensibleLoadManager {
                     CompletableFuture<Optional<BrokerLookupData>>>newBuilder()
             .build();
 
+    private final Map<String, String> brokerToFailureDomainMap = new HashMap<>();
```

Comment:
After you moved `FailureDomainUpdate` listener to `AntiAffinityGroupPolicyHelper `, can we move this map to helper either?

Reply:
updated